docs/sql-ref/010-SQL-Reference/011-SQL-Functions/score-function-miscellaneous-20e6f8e.md

```diff
@@ -409,21 +409,21 @@ A value greater than 0.0 and less than or equal to 1.0.
 
 ## Syntax 1
 
-The following example creates a table that contains two strings and then inserts values into the table.
+The following example creates a table that contains a string and adds a fuzzy text index to it.
 
 ```
-CREATE TABLE T1 (CONTENT TEXT FAST PREPROCESS OFF);
+CREATE TABLE T1 (CONTENT NCLOB);
```

Comment:
The NCLOB needs MEMORY THRESHOLD NULL or you will get an error when creating the fuzzy search index:
"cannot create fuzzy search index: column CONTENT of type NCLOB, only MEMORY THRESHOLD NULL is supported"